cnchi/features_info.py

```diff
@@ -102,7 +102,8 @@ def _(message):
                  "with a mobile version for Android, by the Mozilla Foundation and\n"
                  "its subsidiary, the Mozilla Corporation. Firefox uses the Gecko\n"
                  "layout engine to render web pages, which implements current and\n"
-                 "anticipated web standards."),
+                 "anticipated web standards.  If Firefox is not chosen, Cnchi will\n"
+				 "be installing chromium."),
```

Comment:
@Wyn10 let's change this to `Enable this option to install Firefox instead of Chromium (the default).`